compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -688,14 +688,16 @@ class Allocator {
     Quarantine.disable();
     Primary.disable();
     Secondary.disable();
-    if (Depot)
-      Depot->disable();
+    AllocationRingBuffer *RB = getRingBuffer();
+    if (RB)
+      RB->disable();
```

Comment:
Ah, sorry my bad,

I want to say something like adding two functions, `DisableRingBuffer`/`EnableRingBuffer`

So that we don't lift the details of ring buffer to the user

Reply:
I don't think this matters either way, but sure.